src/android-native/src/main/java/io/liteglue/SQLitePlugin.java

```diff
@@ -52,14 +52,12 @@
     static SQLiteConnector connector = new SQLiteConnector();
 
     static String LOG_TAG = SQLitePlugin.class.getSimpleName();
-
-    protected Activity activity = null;
     protected ExecutorService threadPool;
+    private Context context;
 
-    public SQLitePlugin(ReactApplicationContext reactContext, Activity activity) {
+    public SQLitePlugin(ReactApplicationContext reactContext) {
```

Comment:
no incompatibility as everyone would use Package, there is no need to create Module directly, right?